microshift_troubleshooting/microshift-troubleshoot-backup-restore.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="microshift-troubleshoot-backup-restore.adoc"]
+= Troubleshoot data backup and restore
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-troubleshoot-data-backup-and-restore
+
+toc::[]
+
+To troubleshoot failed data backups and restorations, check the basics first, such as data paths, storage configuration, and storage capacity.
+
+[id="troubleshoot-backup-restore-microshift-backup-data-failed"]
+== Backup data failed
+Data backups are automatic on `rpm-ostree` systems. If you are not using an `rpm-ostree` system and attempted to create a manual backup, the following reasons could cause the backup to fail:
```

Comment:
```suggestion
Data backups are automatic on `rpm-ostree` systems. If you are not using an `rpm-ostree` system and attempted to create a manual backup, the following reasons can cause the backup to fail:
```
From the IBM Style Word Usage page: Do not use _could_ in the present tense to mean _might_ or _can_. Use only in the past conditional form, for example, “files that could not be identified”.